> I've added @ClaudiaLSS and @mohammadamint because I believe they may also be on Windows or a non-Linux system and may help test this.

I have some issues running it (get some errors), not sure if it is the correct way or not:

```
from ramp import ramp_py2xlsx

ramp_py2xlsx(
    "ramp/example/input_file_1.py", # using the example files in ramp
    output_path="test.xlsx"
)


---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
File c:\Users\TAHAVORM\Downloads\GitRepos\RAMP\amin_test.py:2
      [1](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/amin_test.py:1) # %%
----> [2](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/amin_test.py:2) ramp_py2xlsx(
      [3](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/amin_test.py:3)     "ramp/example/input_file_1.py",
      [4](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/amin_test.py:4)     output_path="test.xslx"
      [5](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/amin_test.py:5) )

File c:\Users\TAHAVORM\Downloads\GitRepos\RAMP\ramp\ramp_convert_old_input_files.py:84, in convert_old_user_input_file(fname_path, output_path, suffix, keep_names)
     [81](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/ramp/ramp_convert_old_input_files.py:81) output_fname = fname_path.split(os.path.sep)[-1].replace(".py", suffix)
     [82](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/ramp/ramp_convert_old_input_files.py:82) output_fname = os.path.join(output_path, output_fname)
---> [84](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/ramp/ramp_convert_old_input_files.py:84) file_module = importlib.import_module(fname)
     [86](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/ramp/ramp_convert_old_input_files.py:86) user_list = file_module.User_list
     [88](file:///C:/Users/TAHAVORM/Downloads/GitRepos/RAMP/ramp/ramp_convert_old_input_files.py:88) UseCase(users=user_list).save(output_fname)

File c:\Users\TAHAVORM\AppData\Local\miniconda3\envs\ramp\lib\importlib\__init__.py:127, in import_module(name, package)
    [125](file:///C:/Users/TAHAVORM/AppData/Local/miniconda3/envs/ramp/lib/importlib/__init__.py:125)             break
    [126](file:///C:/Users/TAHAVORM/AppData/Local/miniconda3/envs/ramp/lib/importlib/__init__.py:126)         level += 1
--> [127](file:///C:/Users/TAHAVORM/AppData/Local/miniconda3/envs/ramp/lib/importlib/__init__.py:127) return _bootstrap._gcd_import(name[level:], package, level)

File <frozen importlib._bootstrap>:1030, in _gcd_import(name, package, level)

File <frozen importlib._bootstrap>:1007, in _find_and_load(name, import_)

File <frozen importlib._bootstrap>:984, in _find_and_load_unlocked(name, import_)

ModuleNotFoundError: No module named 'ramp/example/input_file_1'


```

--- a/ramp/__init__.py
+++ b/ramp/__init__.py
@@ -18,6 +18,7 @@ from ramp.core.core import UseCase, User, Appliance
 from ramp.core.utils import yearly_pattern, get_day_type
 from ramp.example.examples import load_data, download_example
 from ramp.post_process.post_process import Plot
+from ramp.ramp_convert_old_input_files import convert_old_user_input_file as ramp_py2xlsx
 
 __authors__ = "Listed in AUTHORS"
 __copyright__ = (

--- a/ramp/ramp_convert_old_input_files.py
+++ b/ramp/ramp_convert_old_input_files.py
@@ -8,7 +8,7 @@ from ramp.core.core import UseCase
 
 parser = argparse.ArgumentParser(
     prog="python ramp_convert_old_input_files.py",
-    description="Convert old python input files to xlsx ones",
+    description="Convert RAMP python input files to xlsx ones",
 )
 parser.add_argument(
     "-i",
@@ -88,7 +88,7 @@ def convert_old_user_input_file(
     UseCase(users=user_list).save(output_fname)
 
 
-if __name__ == "__main__":
+def cli():
     args = vars(parser.parse_args())
     fname = args["fname_path"]
     output_path = args.get("output_path")
@@ -105,3 +105,6 @@ if __name__ == "__main__":
                 )
         else:
             convert_old_user_input_file(fname, output_path=output_path, suffix=suffix)
+
+if __name__ == "__main__":
+    cli()

--- a/ramp/ramp_convert_old_input_files.py
+++ b/ramp/ramp_convert_old_input_files.py
@@ -82,7 +82,7 @@ def convert_old_user_input_file(
     output_fname = os.path.join(output_path, output_fname)
 
     if os.name != "posix":
-        sys.path.insert(0, os.path.abspath(fname_path))
+        sys.path.insert(0, os.path.dirname(os.path.abspath(fname_path)))
     file_module = importlib.import_module(fname)
 
     user_list = file_module.User_list

--- a/ramp/ramp_convert_old_input_files.py
+++ b/ramp/ramp_convert_old_input_files.py
@@ -81,6 +81,8 @@ def convert_old_user_input_file(
     output_fname = fname_path.split(os.path.sep)[-1].replace(".py", suffix)
     output_fname = os.path.join(output_path, output_fname)
 
+    if os.name != "posix":
+        sys.path.insert(0, os.path.abspath(fname_path))
     file_module = importlib.import_module(fname)
 
     user_list = file_module.User_list

--- a/setup.py
+++ b/setup.py
@@ -39,6 +39,7 @@ setup(
     entry_points={
         "console_scripts": [
             "ramp=ramp.cli:main",
+            "ramp_convert=ramp.ramp_convert_old_input_files:cli",
         ],
     },
 )

